api/pkg/db/migration/migration.go

```diff
@@ -59,6 +60,10 @@ func Migrate(api *app.APIBase) error {
 			&model.SyncJob{},
 			&model.Scope{},
 			&model.Config{},
+			&model.Platform{},
```

Comment:
why are we adding this now here?

Reply:
when we create new DB then `platforms` table was not getting created. Hence added this